Makefile

```diff
@@ -20,6 +20,10 @@ show-coverage:
 e2e-tests:
 	behave --tags=-skip
 
+e2e-tests-fast: export BEHAVE_LOCAL_ENV=TRUE
+e2e-tests-fast:
+	behave --tags=-skip --no-capture
```

Comment:
We can also get rid of the --tags=-skip part there's no skip tagged tests anymore